### ISSUE TITLE
Is there a chance of getting flagged as a false positive with another anti cheat?

### ISSUE THREAD

**Issue by Gitfelc** [EQAditu/AdvancedCombatTracker#38]
I couldn't make an account on the website (registration kept loading endlessly) so posting this here, sorry in advance.

This has been posted in the [forums](https://forums.advancedcombattracker.com/discussion/204/can-act-get-me-banned-in-other-games) before but it would be great if it could be clarified what ACT is accessing with all the permissions it is getting. As an example: Valve says that their anti cheat reads if the memory of the game is being accessed or if anything is being injected into the process. If I play CS:GO with ACT open is there a chance that VAC might flag it as false positive, or is ACT only accessing the Final Fantasy process and networking directly? I'm a complete noob programming wise so having a hard time formulating everything so hopefully it's still understandable.

To dumb it down even more:
Cheatengine only triggers a VAC ban if it's hooked into the game directly. Does ACT hook into anything (be it network or the process directly) that isn't Final Fantasy?

Again sorry for posting this here but it would make sleeping at night easier to know if there is any risk when forgetting to close ACT before starting up another game. Obviously Valve, Blizzard or whoever can decide to ban you for whatever they want to so this is more meant as a "does ACT do anything to get flagged" question.

**Comment by EQAditu** [comment]
ACT reads log files.  Nothing more.

Other plugins you load into ACT does whatever those things do.  Most games actually produce usable log files, so it isn't anything additional.  FFXIV doesn't provide usable log files, so the parsing plugin has to make its own.  It does this by reading game memory if the EXE name matches and reading the network traffic of the processes it reads the memory of.  Most plugin authors refuse to make any functionality that modifies the game in any way.  But plugins can do whatever they want.  It's up to you to understand what those plugins do before loading them into ACT.  I cannot give you a blanket answer.

Please realize that things like FRAPS, Riva Tuner Statistics Server, Steam, Discord, Logitech Hub all directly inject code into pretty much any DirectX program.  For the most part, ACT does not.  You must be responsible for not using a plugin that will.  ACT does not and cannot blacklist anything a plugin wants to do.
